README.md

```diff
@@ -122,14 +123,22 @@ nextflow run phac-nml/measeq \
     --input <SAMPLESHEET> \
     --outdir <OUTDIR> \
     --reference <REFERENCE FASTA> \
-    --platform <illumina||nanopore> \
-    --model <MODEL> \
+    --platform nanopore \
+    --model <CLAIR3_MODEL> \
     -profile <docker/singularity/institute/etc>
 ```
 
+#### Clair3 Models
+
+The Nanopore pipeline utilizes [Clair3](https://github.com/HKU-BAL/Clair3) to call nanopore variants which requires a model that should be picked based off of the flowcell, pore, translocation speed, and basecalling model.
+
+Some models are built into clair3 and some need to be downloaded. The [pre-trained clair3](https://github.com/HKU-BAL/Clair3?tab=readme-ov-file#pre-trained-models) models are able to be automatically downloaded when running the pipeline using [`artic get_models`](https://github.com/artic-network/fieldbioinformatics/blob/master/artic/get_models.py) and can be specified as a parameter with `--model <MODEL>`.
+
+Additional or local models can also be used, you just have to provide a path to them and use the `--local_model <PATH>` parameter instead
```

Comment:
As a note, for IRIDA Next I'd recommend only ever using `--local_model` (which we pre-install) since I would like to minimize dependencies on remote services each time the pipeline runs. But this isn't relevant for this PR.

Reply:
Thanks. No need to hide it in this PR.